pkg/features/embedded/immutable-rootfs/usr/lib/dracut/modules.d/30elemental-immutable-rootfs/parse-elemental-cmdline.sh

```diff
@@ -43,7 +43,7 @@ esac
 
 info "root device set to root=${root}"
 
-wait_for_dev -n "${root}"
+wait_for_dev -n "${root#block:}"
```

Comment:
I assume the change is backward compatible, I mean it will also work on old dracut versions, right?

Reply:
Yes, it will only remove block: prefix if it exists. :+1: